server/routes/api/parents/distanceApi.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+
+const apiKey = 'AnRsPfslZKL-b7n6v3v3XsM3yz2lDBemqmkFfMm31zy9xvvbIUTIVDLvX1lKoHoq';
```

Comment:
If we plan to use this value from an env_var (.env) we need to make sure we can provide the details to Lior on how to use it and also mask it in the tests, I can help if needed